client/homebrew/editor/editor.jsx

```diff
@@ -82,11 +83,49 @@ const Editor = createClass({
 		if(!this.refs.codeEditor) return;
```

Comment:
Method should probably be renamed, because it now does more than just highlighting of `\page` lines

Reply:
Done.